tests/custom_maven_install/regression_testing_install.json

```diff
@@ -5816,6 +5816,209 @@
+        "io.undertow.xnio.client.http2.Http2ClientProvider",
+        "io.undertow.xnio.client.http2.Http2PriorKnowledgeClientProvider"
+      ]
+    },
+    "io.quarkus.http:quarkus-http-servlet": {
+      "io.undertow.attribute.ExchangeAttributeBuilder": [
+        "io.undertow.servlet.attribute.ServletContextAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletNameAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRelativePathAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestCharacterEncodingAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestLineAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestLocaleAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestParameterAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestURLAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestedSessionIdAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestedSessionIdFromCookieAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletRequestedSessionIdValidAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletSessionAttribute$Builder",
+        "io.undertow.servlet.attribute.ServletSessionIdAttribute$Builder"
+      ],
+      "io.undertow.predicate.PredicateBuilder": [
+        "io.undertow.servlet.predicate.DirectoryPredicate$Builder",
+        "io.undertow.servlet.predicate.DispatcherTypePredicate$Builder",
+        "io.undertow.servlet.predicate.FilePredicate$Builder"
+      ],
+      "io.undertow.server.handlers.builder.HandlerBuilder": [
+        "io.undertow.servlet.handlers.MarkSecureHandler$Builder"
+      ]
+    },
+    "io.sundr:builder-annotations": {
+      "javax.annotation.processing.Processor": [
+        "io.sundr.builder.internal.processor.BuildableProcessor",
+        "io.sundr.builder.internal.processor.ExternalBuildableProcessor"
+      ]
+    },
+    "io.sundr:resourcecify-annotations": {
+      "javax.annotation.processing.Processor": [
+        "io.sundr.resourcecify.internal.processor.ResourcecifyProcessor"
+      ]
+    },
+    "org.apache.flink:flink-shaded-jackson": {
+      "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.JsonFactory": [
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.JsonFactory",
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.dataformat.csv.CsvFactory",
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.dataformat.yaml.YAMLFactory"
+      ],
+      "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.ObjectCodec": [
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.ObjectMapper"
+      ]
+    },
+    "org.apache.kafka:kafka-clients": {
+      "org.apache.kafka.common.config.provider.ConfigProvider": [
+        " #",
```

Comment:
@illicitonion Is this comment with the license expected output?

Reply:
Ooh, no it's not, will send a PR in a sec!

Reply:
https://github.com/bazelbuild/rules_jvm_external/pull/1101 - thanks for the catch!